conf.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# MbedTLSDocs documentation build configuration file, created by
+# sphinx-quickstart on Wed Jul 21 18:29:51 2021.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['myst_parser']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = ['.rst', '.md']
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'MbedTLSDocs'
```

Comment:
Why this weird StudlyCapsifiedName? Shouldn't the project name be “Mbed TLS”?

Reply:
The name of this project must be distinct from "Mbed TLS" as this will eventually be a subproject of the main Mbed TLS project containing the versioned documentation (since the docs are distributed across 2 separate repos).

I take your point on the CamelCase though - that's my overactive aversion to spaces manifesting itself. I'll fix this in the next one.

Reply:
`project` is a human-readable name, not an internal identifier. It should be spelled the way a human would spell it, with spaces where warranted.